docs(GcpNfsBackupSchedule): user documentation (#614)
kyma-project/cloud-manager: docs/user/tutorials/01-80-gcp-scheduled-nfs-backup.md

> # Create scheduled automatic RWX Volume backups in GCP

Reviewer: ```suggestion
# Create Scheduled Automatic RWX Volume Backups in GCP
```

> # Create scheduled automatic RWX Volume backups in GCP

Reviewer 2: ```suggestion
# Create Scheduled Automatic RWX Volume Backups in Google Cloud
```

> This tutorial explains how to create scheduled automatic backups for ReadWriteMany (RWX) volumes in Google Cloud Platform (GCP).

Reviewer 2: ```suggestion
This tutorial explains how to create scheduled automatic backups for ReadWriteMany (RWX) volumes in Google Cloud (GCP).
```

> ## Preparation <!-- {docsify-ignore} -->

Reviewer: ```suggestion
## Prerequisites <!-- {docsify-ignore} -->
```

> Create a GcpNfsVolume by referring to [Use RWX Volumes in GCP](./01-20-gcp-nfs-volume.md).

Reviewer: ```suggestion
* You have created a GcpNfsVolume. Follow [Use RWX Volumes in GCP](./01-20-gcp-nfs-volume.md) to learn more.
```

> All the examples below assume that the GcpNfsVolume is named `my-vol` and is in the same namespace as the GcpNfsBackupSchedule resource.

Reviewer: ```suggestion
> [!NOTE]
> All the examples below assume that the GcpNfsVolume is named `my-vol` and is in the same namespace as the GcpNfsBackupSchedule resource.
```

## Steps <!-- {docsify-ignore} -->

1. Export the namespace as an environment variable. Run:

   ```shell
   export NAMESPACE={NAMESPACE_NAME}
   ```

2. Create a GcpNfsBackupSchedule resource.

   ```shell
   cat <<EOF | kubectl -n $NAMESPACE apply -f -
   apiVersion: cloud-resources.kyma-project.io/v1beta1
   kind: GcpNfsBackupSchedule
   metadata:
      name: my-backup-schedule
   spec:
      nfsVolumeRef:
        name: my-vol
      schedule: "0 * * * *"
      prefix: my-hourly-backup
      deleteCascade: true
   EOF
   ```

3. Wait for the GcpNfsVolumeBackup to be in the `Active` state.
   ```shell
   kubectl -n $NAMESPACE wait --for=jsonpath='{.status.state}'=Active gcpnfsbackupschedule/my-backup-schedule --timeout=300s
   ```
   Once the GcpNfsVolumeBackup is created, you should see the following message:
   ```
   gcpnfsbackupschedule.cloud-resources.kyma-project.io/my-backup-schedule condition met
   ```

> 4. Observe the nextRunTimes for the creating the backups.

Reviewer: ```suggestion
4. Observe the nextRunTimes for creating the backups.
```

   ```shell
   kubectl -n $NAMESPACE get gcpnfsbackupschedule my-backup-schedule -o jsonpath='{.status.nextRunTimes}{"\n"}' 
   ```

> 5. Wait till the time specified in the nextRunTimes (in previous step) and see that the GcpNfsVolumeBackup objects get created.

Reviewer: ```suggestion
5. Wait till the time specified in the nextRunTimes (in the previous step) and see that the GcpNfsVolumeBackup objects get created.
```
